content/en/blog/_posts/2023-08-29-Gateway-API-v080.md

```diff
@@ -0,0 +1,176 @@
+Gateway API v0.8.0 CRDs include CEL validation and cannot be installed on these
+versions of Kubernetes. Note that Gateway API only commits to providing support
+for the [5 most recent versions of
+Kubernetes](https://gateway-api.sigs.k8s.io/concepts/versioning/#supported-versions),
+and thus these versions are no longer supported by Gateway API.
+
+## How can I get started with the Gateway API?
+
+Gateway API represents the future of load balancing, routing, and service mesh
+APIs in Kubernetes. There are already more than 20 implementations[impl]
+available, and the list keeps growing. Because this is a CRD-based API, you can
+install the latest version on any Kubernetes 1.23+ cluster.
+
+If you're interested in getting started with Gateway API, take a look at the
+[API concepts documentation][concepts] and check out some of the
+[Guides][guides] to try it out.
+
+If you're specifically interested in helping to contribute to Gateway API, we
+would love to have you! Please feel free to [open a new issue][issue] on the
+repository, or join in the [discussions][disc]. Also check out the [community
+page][community] which includes links to the Slack channel and community
+meetings. We look forward to seeing you!!
+
+## Further Reading
+
+If you're interested in learning more about Service Mesh integration in Gateway
+API, it's worth reading the enhancement proposals (GEPs) that got us to this
+point:
+
+- [GEP-1324](https://gateway-api.sigs.k8s.io/geps/gep-1324/) provides an
+  overview of the GAMMA goals and some important definitions. This GEP is well
+  worth a read for its discussion of the problem space.
+- [GEP-1426](https://gateway-api.sigs.k8s.io/geps/gep-1426/) defines how to
+  use Gateway API route resources, such as HTTPRoute, to manage traffic within
+  a service mesh.
+- [GEP-1686](https://gateway-api.sigs.k8s.io/geps/gep-1686/) builds on the
+  work of [GEP-1709](https://gateway-api.sigs.k8s.io/geps/gep-1709/) to define
+  a _conformance profile_ for service meshes to be declared conformant with
+  the Gateway API.
+
+Although these are [Experimental][status] APIs, note that they are available
+in the [`standard` release channel][ch], since the GAMMA initiative has not
+needed to introduce new resources or fields to date.
+
+[gamma]:/concepts/gamma/
+[status]:/geps/overview/#status
+[ch]:/concepts/versioning/#release-channels-eg-experimental-standard
+[crd]:https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/
+[concepts]:/concepts/api-overview/
+[guides]:/guides/getting-started/
+[impl]:/implementations/
+[install-crds]:/guides/getting-started/#install-the-crds
+[issue]:https://github.com/kubernetes-sigs/gateway-api/issues/new/choose
+[disc]:https://github.com/kubernetes-sigs/gateway-api/discussions
+[community]:/contributing/community/
+[mesh-routing]:/concepts/gamma/#how-the-gateway-api-works-for-service-mesh
+[GEP-1426]:/geps/gep-1426/
```

Comment:
These relative links don't work. They need to change to be https://gateway-api.sigs.k8s.io/ absolute links, and the https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/ link should switch to a relative link.

Reply:
Oh ouch, you're right.

Reply:
Yep, sorry, that's on me. Should be updated across the board now and have manually verified each link. They all should work with the exception of the release link which will work once a release is published.